components/library/http-parser/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Denys Rtveliashvili.  All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		http-parser
+COMPONENT_VERSION=	2.1
+COMPONENT_FMRI=		library/http-parser
+COMPONENT_CLASSIFICATION=System/Libraries
+COMPONENT_PROJECT_URL=	https://packages.qa.debian.org/h/http-parser.html
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:749c35dc4696be4be9bb4f3b388197151cee6e91b5e8895789be904386e38187
+COMPONENT_ARCHIVE_URL=	http://http.debian.net/debian/pool/main/h/http-parser/http-parser_2.1.orig.tar.gz
```

Comment:
There's already 2.7.0 release (https://github.com/nodejs/http-parser/releases), why do you want so old package?